media-tv/plex-media-server/plex-media-server-1.4.0.3224.ebuild

```diff
@@ -27,11 +26,11 @@ KEYWORDS="-* ~amd64"
 
 IUSE="plex-ffmpeg"
 
-DEPEND="
-	net-dns/avahi
-	sys-apps/fix-gnustack"
+DEPEND="sys-apps/fix-gnustack
+	dev-python/virtualenv[${PYTHON_USEDEP}]"
 
-RDEPEND="plex-ffmpeg? ( >=media-video/plex-ffmpeg-2016.12 )"
+RDEPEND="plex-ffmpeg? ( >=media-video/plex-ffmpeg-2016.12 )
+	net-dns/avahi"
```

Comment:
Is there any reason the Avahi USE flag isn't used here? I'm happy to add it myself; I'm just wondering if there was a reason it was omitted.